test/extensions/filters/http/cache/http_cache_test.cc

```diff
@@ -170,21 +170,66 @@ TEST_F(LookupRequestTest, NotExpiredViaFallbackheader) {
   EXPECT_EQ(CacheEntryStatus::Ok, lookup_response.cache_entry_status_);
 }
 
-TEST_F(LookupRequestTest, FullRange) {
-  request_headers_.addCopy("Range", "0-99");
+TEST_F(LookupRequestTest, SatisfiableRange) {
+  // add method (GET) and range to headers
+  request_headers_.addCopy(Http::Headers::get().Method.get(),
```

Comment:
I think these can both be addReference, as the keys and values will live beyond the map.

similar patterns below.